nautobot/ipam/tests/test_views.py

```diff
@@ -1006,7 +1007,6 @@ class VLANTestCase(ViewTestCases.PrimaryObjectViewTestCase):
     @classmethod
     def setUpTestData(cls):
```

Comment:
This test changeset also probably backports to `develop`. I'll see about getting a PR open after merging this.